server/api/jellyfin.ts

```diff
@@ -417,11 +417,28 @@ class JellyfinAPI extends ExternalAPI {
 
   public async getEpisodes(
     seriesID: string,
-    seasonID: string
-  ): Promise<JellyfinLibraryItem[]> {
+    seasonID: string,
+    options: { includeMediaInfo: true }
+  ): Promise<JellyfinLibraryItemExtended[]>;
+  public async getEpisodes(
+    seriesID: string,
+    seasonID: string,
+    options?: { includeMediaInfo?: false }
+  ): Promise<JellyfinLibraryItem[]>;
+  public async getEpisodes(
+    seriesID: string,
+    seasonID: string,
+    options?: { includeMediaInfo?: boolean }
+  ): Promise<JellyfinLibraryItem[] | JellyfinLibraryItemExtended[]> {
```

Comment:
I find something like this better and less verbose:
```suggestion
public async getEpisodes<T extends { includeMediaInfo?: boolean; }>(
  seriesID: string,
  seasonID: string,
  options?: T
): Promise<EpisodeReturn<T>> {
```
With `EpisodeReturn` like this:
```ts
type EpisodeReturn<T> = T extends { includeMediaInfo: true }
  ? JellyfinLibraryItemExtended[]
  : JellyfinLibraryItem[];
```